surefire-its/src/test/java/org/apache/maven/surefire/its/UmlautDirIT.java

```diff
@@ -112,19 +114,29 @@ public void surefire1617WithColonInLocalRepo()
                 .executeTest()
                 .verifyErrorFreeLog()
                 .assertTestSuiteResults( 1, 0, 0, 0 );
+
+        String unproblematicDirName = "surefire-1617-colon-in-local-repo_renamed-after-test";
+        Path localRepoDirWithUnproblematicName = localRepoDirWithColonInName.resolveSibling( unproblematicDirName );
+        Files.move( localRepoDirWithColonInName, localRepoDirWithUnproblematicName );
```

Comment:
We should put it in a final section to allow works also when test fails

Reply:
Done.